### PR TITLE
docs: Update local-development.md

### PR DESCRIPTION
**Pull request by infraredgirl**
- Remove incorrect mention of number of steps
- Remove unnecessary `rm` command, since `>` redirect will already remove the file
- Correct debugging instructions

### CODE REVIEW

**Comment by rarkins** [comment]
The --log-level=debug param is incorrect too. We switched to env only (LOG_LEVEL=debug) so that CLI parsing can be debug logged too!

**Comment by infraredgirl** [comment]
> The --log-level=debug param is incorrect too. We switched to env only (LOG_LEVEL=debug) so that CLI parsing can be debug logged too!

Thanks for the pointer - I'll make an update to address for this!

File: docs/development/local-development.md [hunk 1]
```diff
@@ -181,10 +181,10 @@ It's usually easier to have the logs in a file that you can open with a text edi
 You can use a command like this to put the log messages in a file:
 
 ```
-rm -f debug.log && yarn start myaccount/therepo --log-level=debug > debug.log
+LOG_LEVEL=debug yarn start myaccount/therepo > debug.log
 ```
 
-The example command will delete any existing `debug.log` and then save Renovate's output to a new `debug.log` file.
+The example command save Renovate's output to the `debug.log` file (and overwrite `debug.log` if it already exists).
```
Comment (rarkins) [inline docs/development/local-development.md:187]:
```suggestion
The example command will redirect/save Renovate's output to the `debug.log` file (and overwrite `debug.log` if it already exists).
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/development/local-development.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ To ensure everything is working properly on your end, you must:
 1. Verify all tests pass and have 100% test coverage, by running `yarn test`
 1. Verify the installation by running `yarn start`. You must see this error: `You must configure a GitHub personal access token`
 
-You only need to do these 5 steps this one time.
+You only need to do these steps this one time.
```
Comment (JamieMagee) [inline docs/development/local-development.md:83]:
```suggestion
You only need to do these steps once.
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**